### ISSUE TITLE
EOF Highlighting Error

### ISSUE THREAD

**Issue by petems** [GiantToast/language-terraform#35]
The following code does not break the EOF highlighting properly:

```hcl
policy = <<EOF
  {
    "Version": "2008-10-17",
    "Statement": [
    {
      "Sid": "PublicReadForGetBucketObjects",
      "Effect": "Allow",
      "Principal": {
        "AWS": "*"
        },
        "Action": "s3:GetObject",
        "Resource": "arn:aws:s3:::${var.bucket_name}/*"
      }
      ]
    }
    EOF
}

resource "aws_cloudfront_distribution" "distribution" {
```

## Wrong Highlighting
![screenshot1](https://user-images.githubusercontent.com/1064715/36945789-14e8d64c-1f81-11e8-997c-f499970387ed.png)

#### Adding a tab seems to fix highlighting

![screenshot2](https://user-images.githubusercontent.com/1064715/36945790-14f43e24-1f81-11e8-9c77-8143b32ccdd4.png)

**Comment by cmur2** [comment]
Is this still an issue in v0.9.3 of this package?

**Comment by lindeskar** [comment]
Seems to still be an issue in v0.9.7
![image](https://user-images.githubusercontent.com/18196573/159714917-90ab3435-1b70-46ed-8ef9-a089198ebe3d.png)

**Comment by cmur2** [comment]
I can relate, I recently saw this when the final `EOF` is not at the beginning of the line.

Reading https://www.terraform.io/language/expressions/strings#indented-heredocs I learned that for `<<EOF` this is expected. There is an alternative syntax of indented `<<-EOF` available, did you try that?

**Comment by lindeskar** [comment]
Yep, changing to the alternative syntax works, just as removing the indented end marker.

I'll run a `sed` on our files to fix. Thanks!
